server.py

```diff
@@ -41,11 +41,24 @@ def create_application():
 @APP.route(f'{ROUTE_PREFIX}/', methods=['GET'], strict_slashes=False)
 def get_root():
     """Root Endpoint for 3scale."""
+    if not collector.WORKER:
+        return_code = 500
+        status = 'Error'
+        message = 'No worker set'
+    elif not collector.utils.ping_redis():
+        return_code = 500
+        status = 'Error'
+        message = 'Required service not operational'
```

Comment:
since we are specifically testing for `redis`, shall we make it clear in the message?

Reply:
Want to keep this generic because, in the next round (in the next iteration) I also want to check for availability of the AI Service.

Currently none of our AI Services have a route to check for liveness, so those routes would have to be implemented first, before I can use them here

Reply:
sounds good